components/post_view/failed_post_options/index.js

```diff
@@ -15,9 +17,12 @@ function mapStateToProps(state, ownProps) {
 
 function mapDispatchToProps(dispatch) {
     return {
-        actions: bindActionCreators({
-            removePost,
-        }, dispatch),
+        actions: {
+            ...bindActionCreators({
+                removePost,
+            }, dispatch),
+            createPost,
```

Comment:
I haven't made `createPost` into an dispatchable action, but just wanted to mock it for unit testing.